### DISCUSSION TITLE
Branch Name gets replaced accidentally?

### DISCUSSION THREAD

**Discussion by mstoetzer** [renovatebot/renovate#21699] [Suggest an Idea]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

27.6.4

### Describe the bug

Using the following package rule adds `PROD ` prefix to the commit message and the Mergerequest title

```json
    {
      "description": "Create MR for PROD dependencies, when releases are stable for 5 days",
      "matchManagers": ["kubernetes"],
      "matchPaths": ["prod/**"],
      "additionalBranchPrefix": "prod/",
      "commitMessagePrefix": "PROD",
      "stabilityDays": 5,
      "prCreation": "not-pending"
    },

```

Example: `PROD Update prom/prometheus Docker tag to v2.30.0`

The dependency dashboard shows some `MROD  Update prom/prometheus Docker tag to v2.30.0` in the list of merge request:

![Screenshot from 2021-09-16 11-37-04](https://user-images.githubusercontent.com/4034031/133588997-c07f7bee-8ec7-487f-b4da-807c42e9a899.png)

Could it be, that somehow `PR` gets replaced by `MR` because its gitlab and not github, that renovate is running for?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
Yes, PR gets replaced by MR :)

**Comment by mstoetzer** [comment]
This also applies for Mergerequest descriptions:

e.g.

```
[IMMROVEMENT] refactor engine-image-controller to use structured logging
```

:)
